src/traversal/__tests__/writeRelayUpdatePayload-test.js

```diff
@@ -1078,6 +1078,196 @@ describe('writeRelayUpdatePayload()', () => {
       // connection metadata is merged into the queued store
       expect(store.getField(connectionID, 'count')).toBe(0);
     });
+
+    // test cases for array path deletedIDFieldName
```

Comment:
This comment seems redundant. If you wanted to get rid of the repeated references to "array path deletedIDFieldName" below you could use a nested `describe`:

```
describe('with an array path `deletedIDFieldName`', () => {
  it('optimistically deletes comments, () => { /* ... */ });
  it('non-optimistically deletes comments, () => { /* ... */ });
});
```